charts/rancher-istio/1.1.5-rancher1/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: rancher-istio
+version: 0.0.1
```

Comment:
we should change this version to `1.1.5-rancher1`

Reply:
done.